app/components/avatars/avatars.stories.tsx

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React from 'react';
+
+import {storiesOf} from '@storybook/react-native';
+import {withKnobs, array, number, select} from '@storybook/addon-knobs';
+import {Preferences} from '@mm-redux/constants';
+
+import Avatars from './avatars';
+
+storiesOf('Avatars', module).
+    addDecorator(withKnobs).
+    add('Avatars', () => (
+        <Avatars
+            size={number('size', 24, {range: true, min: 16, max: 80, step: 8})}
+            userIds={array('userIds', ['cb5wff5ah3y8ifijsgwrdhnn9o', 'psd3g9p9sjykuxqijh9qzmh86w', 'ue6q4zzgntykikm6f6fy5tju8r', 'iq164nrrmjff5mtbafr4jpukcy'], ',')}
```

Comment:
These userIds are specific to my test server. You can change these with userIds from your test server. UserIds are viewable in the "User Management" section of the system console.

Reply:
Please try and find a way not to hard code this ID's

Reply:
@enahum 

I’ve been bashing my head against a wall for a week trying to figure this out. I’ve been trying to dynamically download the user data from the server.

1. Storybook react-native doesn’t support doing some async work like fetching of data via external API before rendering/adding a story. 
Storybook react (i.e. for the web) only just received support for doing this in version 6.0 (which was released on August). Storybook for react-native still does not support it. The way around this is to use a wrapper component, see this suggestion: https://github.com/storybookjs/storybook/issues/696#issuecomment-280368818

2. Promises seem to be broken in storybook for react-native.  https://github.com/storybookjs/react-native/issues/57
The fix for this is very hacky. This makes it impossible to use our configured client (`Client4`) to download any data from the server. 

At this point I’m not sure what to try next.

Reply:
but you added the ability to get data from the redux store, yes? so you should  be able to get user profiles that you already have stored there, right ?

Reply:
Indeed, I was so fixated about getting the client to work that I forgot about the state. I've updated to use the state. I'll update the PR summary